monorepo-folders/packages/temporal-workflows/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "temporal-workflows",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "compile": "tsc --build",
+    "lint": "eslint ."
+  },
+  "nodemonConfig": {
```

Comment:
Unused?

Reply:
yeah i copied and pasted a lot.. need to clean up